### DISCUSSION TITLE
Remove super user

### DISCUSSION THREAD

**Discussion by HazemMohamed252** [statamic/cms#6635] [Q&A]
I wanna to remove super user check from permission. coz i wanna to grant admin all permissions except the super user. I think this need to be handled from the statamic code itself. Please i need this change ASAP. Thanks in advance.

**Comment by edalzell** [comment]
What do you mean "remove super user check"?

The ability to have a super user is built in to Statamic.

**Reply by HazemMohamed252** [reply]
I mean when i am going to grand user some permissions i want to grant all except to be super user. This feature will be great to be handled

**Reply by jackmcdade** [reply]
You can already do that – just check all the permissions except for Super.

**Reply by HazemMohamed252** [reply]
No, i want to grant all permissions for user except to be SUPER, what happens now when i grant him all he can mark on super user and see all features
![InkedCapture](https://user-images.githubusercontent.com/29832976/190144957-5de45f22-be30-44a1-a2d5-be3c28b35cd0.jpg)

**Reply by HazemMohamed252** [reply]
I suggest from my side that you can make update on statamic and let the user can grant all permissions execpt the super. I wish you could understabd me.  Thanks

**Reply by HazemMohamed252** [reply]
Another way to explain i want to make someone like an admin and have access to create users. groups and permissions but without being SUPER USER

**Reply by duncanmcclean** [reply]
Then just check all the checkboxes and save the role.

**Reply by HazemMohamed252** [reply]
I did that but this user can view super user check and can mark on it. This is an issue

**Comment by jackmcdade** [comment]
Just don’t give them the ability edit roles and you’ll be good.

**Reply by HazemMohamed252** [reply]
I understand your advice but from my opinion as i am developer also it will be good thing if you make this feature that you can grant somesone all permissions for example if i wanna to make group call admin and make permission call also admin and want to grant him access for permissions so he can create, edit and grant some permissions to users BUT i want this admin NOT to be able to see super user check. I wish you understand me

**Reply by jackmcdade** [reply]
I now understand what you're asking for. We don't have plans to add that level of granularity at this time, I'm sorry.

**Reply by HazemMohamed252** [reply]
It's okay but i wish this feature be applied soon it will be great. Thanks in advance.
